### PR TITLE
Feature/56 cubeunits

### LINKED ISSUE #56: TRIPCUBE: proper units!
- DosCubes (.dos suffix) must be in %% as integers.
- LETCubes (.dosemlet.dos suffix) must be in keV/um

### PR DESCRIPTION
**Pull request by nbassler**
Seems to work, output cubes .dos and .dosemlet.dos have proper sizes at least.
This one will resolve #56 and also #60.

### CODE REVIEW

**Comment by nbassler** [comment]
Not sure what to do about the failed checks. @grzanka ?

**Comment by grzanka** [comment]
I've rerun failing checks, probably some issues with build systems.

**Comment by grzanka** [comment]
To make code more PEP8 friendy I usually run: 
```yapf -i -r --style='{COLUMN_LIMIT : 120}' file_or_directory``

Then I inspect if it is OK, this tool sometimes makes stupid things.

yapf - https://github.com/google/yapf

File: pymchelper/writers/trip98.py [hunk 1]
```diff
@@ -8,17 +8,73 @@ def __init__(self, filename):
...
+            cube.cube = detector.data.reshape(detector.nx,
+                                              detector.ny,
+                                              detector.nz)
+            cube.cube = cube.cube * 0.1  # MeV/cm -> keV/um
+
+            cube.write(self.output_corename)
+
+        else:
+            import sys
+            sys.stderr.write("ERROR: can only write dose- or ",
```
Comment (grzanka) [inline pymchelper/writers/trip98.py]: `sys.stderr` should be replaced with `logger.error("Error message")`

**Comment by grzanka** [comment]
Now all tests are passing

**Comment by grzanka** [comment]
PS. I prefer "squash and merge" over normal merge, in case PR has many commits (especially if most of them do not pass tests)

**Pull request merged**